### DISCUSSION TITLE
v0.17.10

### DISCUSSION THREAD

**Discussion by StevenACoffman** [99designs/gqlgen#2240] [Announcements]
## What's Changed
* fix: chat example frontend race condition by @MoofMonkey in https://github.com/99designs/gqlgen/pull/2219
* Update dataloaders.MD by @mimol91 in https://github.com/99designs/gqlgen/pull/2221
* Respect options in WebsocketOnce by @crossworth in https://github.com/99designs/gqlgen/pull/2223
* Config option to only make cyclical struct fields pointers by @ianling in https://github.com/99designs/gqlgen/pull/2174
* Config option to make resolvers return values instead of pointers by @ianling in https://github.com/99designs/gqlgen/pull/2175
* fix #1876: Optional Any type should allow nil values by @vediatoni in https://github.com/99designs/gqlgen/pull/2231
* fix: #2234 Response.Errors in DispatchError function is not PresentedError by @oikyn in https://github.com/99designs/gqlgen/pull/2235
* Use exact capitalization from field names overridden in config by @ianling in https://github.com/99designs/gqlgen/pull/2237
* update gqlparser to v2.4.5 by @StevenACoffman in https://github.com/99designs/gqlgen/pull/2239

## New Contributors
* @mimol91 made their first contribution in https://github.com/99designs/gqlgen/pull/2221
* @crossworth made their first contribution in https://github.com/99designs/gqlgen/pull/2223
* @ianling made their first contribution in https://github.com/99designs/gqlgen/pull/2174
* @vediatoni made their first contribution in https://github.com/99designs/gqlgen/pull/2231
* @oikyn made their first contribution in https://github.com/99designs/gqlgen/pull/2235

**Full Changelog**: https://github.com/99designs/gqlgen/compare/v0.17.9...v0.17.10

<hr /><em>This discussion was created from the release <a href='https://github.com/99designs/gqlgen/releases/tag/v0.17.10'>v0.17.10</a>.</em>

**Comment by skaji** [comment]
@StevenACoffman Thanks for releasing v0.17.10 !

It seems that the PR links in https://github.com/99designs/gqlgen/blob/master/CHANGELOG.md are wrong.
For example, `cac4f404 update gqlparser (#2239)` points to `https://github.com/99designs/gqlgen/pull/39`, not `#2239`.
Could you check it please?

**Reply by StevenACoffman** [reply]
That... does look wrong.  😞 I regenerate it with `git-chglog -o CHANGELOG.md` so this appears to be either a problem with https://github.com/99designs/gqlgen/blob/master/./CHANGELOG-full-history.tpl.md, with https://github.com/99designs/gqlgen/blob/master/.chglog/config.yml or the version of git-chglog I am using.

The changelog entry for the individual releases should be more accurate.
